### ISSUE TITLE
qutebrowser semi-randomly freezes the whole computer.

### ISSUE THREAD

**Issue by eike-fokken** [qutebrowser/qutebrowser#5294]
**Version info**:
qutebrowser v1.10.1
Git commit: 
Backend: QtWebEngine (Chromium 69.0.3497.128)
Qt: 5.12.5

I'm on debian testing with X-org

**Does the bug happen if you start with `--temp-basedir`?**:
Actually, that immediately froze the computer with log message:
[27475:27493:0317/160602.221515:ERROR:nss_util.cc(748)] After loading Root Certs, loaded==false: NSS error code: -8018

**Description**
Starting maybe two to three weeks ago, qutebrowser sometimes freezes, making the computer unusable for a time of usually 30 seconds. I can use the mouse but clicking doesn't do anything. Apparently the keyboard works: The only remedy I found was to switch to a different tty (ctrl + alt +F2) and then manually sending the sigkill signal.

**How to reproduce**

I'm not sure, what triggers this. But for example visiting github sites seems to be a good way to trigger the freeze in maybe on tenth of cases.

I looked through the docs on how to produce relevant data to help finding the cause but am not sure.
As debug symbols are only recommended if I can trigger the bug reliably. Please let me know, what I can do.
I attached my config.py minus two entries that deal with keepassxc
[config.py.txt](https://github.com/qutebrowser/qutebrowser/files/4344202/config.py.txt)

**Comment by eike-fokken** [comment]
Help?

**Comment by toofar** [comment]
Can you try running from a venv to see if it is your system qt? https://www.qutebrowser.org/doc/install.html#_installing_dependencies_including_qt
Also what graphics drivers are you using? Possibly playing with the `qt.force_software_rendering` setting might help.

**Comment by eike-fokken** [comment]
I'm right now trying the venv way. Up to now no crashes.
As for the graphics drivers: How do I find out? I'm on a Thinkpad x260 with debian-testing. The command

```
lspci -k | grep -EA3 'VGA|3D|Display'
```
returns
                      
```00:02.0 VGA compatible controller: Intel Corporation Skylake GT2 [HD Graphics 520] (rev 07)
	Subsystem: Lenovo Skylake GT2 [HD Graphics 520]
	Kernel driver in use: i915
	Kernel modules: i915```

Are these the relevant settings?

**Comment by The-Compiler** [comment]
Given that you're on Intel graphics, this sounds like a duplicate of #5178. Unfortunately, there's nothing qutebrowser can do here, as that's a bug in the Intel graphic drivers. You might want to try switching to the modesetting driver, not sure how to do that on Debian though. On Archlinux, all I needed to do is removing `xf86-video-intel`.

**Comment by eike-fokken** [comment]
Oh, I'm sorry, I didn't find those other issues. Maybe I failed to also search closed issues.
Thank you for the hints!

**Comment by eike-fokken** [comment]
> Can you try running from a venv to see if it is your system qt? https://www.qutebrowser.org/doc/install.html#_installing_dependencies_including_qt

I did this and am now running qutebrowser in this setting. This seems to remove the problem.
Up until now I'm not really inconvenienced by this. I have encountered a video I could not view, probably because of the lack of proprietary codec support.

I guess I'll run this version of qutebrowser until debian testing arrives at this commit and then switch back to the distribution package.
Thanks for the hint!

If you care to speculate: What is the likely cause for the problem, if this remedy works?
